api/users/serializers.py

```diff
@@ -13,18 +37,25 @@ class UserSerializer(JSONAPISerializer):
     ])
     id = ser.CharField(read_only=True, source='_id')
     fullname = ser.CharField(help_text='Display name used in the general user interface')
-    given_name = ser.CharField(help_text='For bibliographic citations')
-    middle_name = ser.CharField(source='middle_names', help_text='For bibliographic citations')
-    family_name = ser.CharField(help_text='For bibliographic citations')
-    suffix = ser.CharField(help_text='For bibliographic citations')
+    given_name = ser.CharField(required=False, allow_blank=True, help_text='For bibliographic citations')
+    middle_name = ser.CharField(required=False, allow_blank=True, source='middle_names', help_text='For bibliographic citations')
+    family_name = ser.CharField(required=False, allow_blank=True, help_text='For bibliographic citations')
+    suffix = ser.CharField(required=False, allow_blank=True, help_text='For bibliographic citations')
     date_registered = ser.DateTimeField(read_only=True)
-    gravatar_url = ser.CharField(help_text='URL for the icon used to identify the user. Relies on http://gravatar.com ')
-    employment_institutions = ser.ListField(source='jobs', help_text='An array of dictionaries representing the '
-                                                                     'places the user has worked')
-    educational_institutions = ser.ListField(source='schools', help_text='An array of dictionaries representing the '
-                                                                         'places the user has attended school')
-    social_accounts = ser.DictField(source='social', help_text='A dictionary of various social media account '
-                                                               'identifiers including an array of user-defined URLs')
+    gravatar_url = ser.CharField(required=False, read_only=True, help_text='URL for the icon used to identify the user. Relies on http://gravatar.com ')
+    employment_institutions = JobsSerializer(read_only=True, required=False, source='jobs', help_text='An array of dictionaries representing the '
```

Comment:
Actually, thinking about it I think we should remove these from the serializer for now. I think we will not want to present these as a simple list, and if we do so now we won't be able to change it. Once we figure out how we want to change the model, or if we for sure determine that we don't, then we can add it into the v2 API. Does that makes sense to you, @sloria?